test-nextjs/convex/auth.ts

```diff
@@ -3,7 +3,7 @@ import GitHub from "@auth/core/providers/github";
 import { convexAuth } from "@convex-dev/auth/server";
 import { internal } from "./_generated/api";
 
-export const { auth, signIn, signOut, store } = convexAuth({
+export const { auth, signIn, signOut, store, isAuthenticated } = convexAuth({
```

Comment:
I think one thing I don't like about this is that it expands the required members of the `auth:` namespace.

Maybe instead I can bundle it in the `auth` object that's already unpacked/exported here?

Reply:
That doesn't seem to work.